samples/spring-security-hybrid-usage/README.md

```diff
@@ -2,10 +2,76 @@
+Follow the deployment steps for [Kyma/Kubernetes](#Deployment-on-Kyma/Kubernetes) or [Cloud Foundry](#Deployment-on-Cloud-Foundry).
+
+# Deployment on Kyma/Kubernetes
+<details>
+<summary>Expand this to follow the deployment steps</summary>
+
+- Build docker image and push to repository
+- Configure the deployment.yml
+- Deploy the application
+- Admin: Assign Role Collection to your XSUAA user
+- Admin: Assign Group to your IAS user
+- Access the application
+
+## Build docker image and push to repository
+```shell script
+mvn spring-boot:build-image -Dspring-boot.build-image.imageName=<repositoryName>/<imageName>
+docker push <repositoryName>/<imageName>
+```
+> This makes use of `Dockerfile`.
+
+## Configure the deployment.yml
+In deployment.yml replace the placeholders:
+- `<YOUR IMAGE REPOSITORY>` with the one created in the previous step.
+
+## Deploy the application
+Deploy the application using [kubectl cli](https://kubernetes.io/docs/reference/kubectl/)
+```shell script
+kubectl apply -f ./k8s/deployment.yml -n <YOUR NAMESPACE>
+```
+
+## Cockpit administration task: Assign Xsuaa Role Collection to your User
+Finally, as part of your Identity Provider, e.g. SAP ID Service, assign the deployed Role Collection `XSUAA-Viewer` to your user as depicted in the screenshot below and as documented [here](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/9e1bf57130ef466e8017eab298b40e5e.html).
+
+![](../images/SAP_CP_Cockpit_AssignRoleCollectionToUser.png)
+
+Further up-to-date information you can get on sap.help.com:
+- [Maintain Role Collections](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/d5f1612d8230448bb6c02a7d9c8ac0d1.html)
+- [Maintain Roles for Applications](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/7596a0bdab4649ac8a6f6721dc72db19.html).
+
+## IAS User administration task: Assign Group to your User
+You need administrator permissions to create a Groups "Read" in IAS and assign it to your user.
+
+## Access the application
+- get an IAS oidc token via ``password`` grant token flow. For that call the ``/oauth2/token`` endpoint of your identity service. You can get the ``url`` and the ``clientid`` and ``clientsecret`` for the Basic Authorization header from the `ias-service-binding` secret.
+    ```shell script
+      kubectl get secret "ias-service-binding" -o go-template='{{range $k,$v := .data}}{{"### "}}{{$k}}{{"\n"}}{{$v|base64decode}}{{"\n\n"}}{{end}}' -n <YOUR NAMESPACE>
+    ```
+- get a XSUAA access token via ``client`` token flow. For that, call the ``/oauth/token`` endpoint of your xsuaa service. <br>You can get the ``url``, the ``client_id``, the ``client_secret`` for the request from the `xsuaa-service-binding` secret.
+    ```shell script
+      kubectl get secret "xsuaa-service-binding" -o go-template='{{range $k,$v := .data}}{{"### "}}{{$k}}{{"\n"}}{{$v|base64decode}}{{"\n\n"}}{{end}}' -n <YOUR NAMESPACE>
+    ```
```

Comment:
i think something is missing here, the way on how to get an token...?

Reply:
do you suggest adding a curl command here for the post request to token endpoint?

Reply:
see also here:
https://github.com/SAP/cloud-security-xsuaa-integration/tree/main/samples/spring-security-hybrid-usage#access-the-application
I think it would be good to clean up our repo and to put the different ways on how to get an access token into a separate README, so that we can link to them as part of our sample READMEs.

Reply:
opened a new issue #652